lib/elixir/lib/process.ex

```diff
@@ -239,10 +239,18 @@ defmodule Process do
   which is not alive or when the given PID exits. Note that timers will not be
   automatically canceled when `dest` is an atom (as the atom resolution is done
   on delivery).
-  """
-  @spec send_after(pid | atom, term, non_neg_integer) :: reference
-  def send_after(dest, msg, time) do
-    :erlang.send_after(time, dest, msg)
+
+  The function accepts one boolean option: `:abs`. When `false` (the default),
+  the `time` is treated as relative to the current monotonic time. When `true`
+  the `time` is an absolute value for the erlang monotonic time when the message
```

Comment:
I don't think we usually use definite form for parameter names? Would it read better if we drop "the" in "the `time`"?